src/test/resources/data/FeedbackRankQuestionUiTest.json

```diff
@@ -1111,6 +1137,58 @@
       },
       "giverSection": "None",
       "recipientSection": "Section 1"
+    },
+    "feedbackResponseForStudentSessionQn16.1": {
+      "feedbackSessionName": "Student Session",
+      "courseId": "FRankUiT.CS4221",
+      "feedbackQuestionId": "16",
+      "feedbackQuestionType": "RANK_RECIPIENTS",
+      "giver": "Team 1</option><option value=\"dump\"></td><td>'\"",
```

Comment:
The giver should be valid email?
Take a look at https://github.com/TEAMMATES/teammates/blob/master/src/test/resources/data/InstructorFeedbackResultsPageUiTest.json#L1475

Reply:
Not when giver is a team ? https://github.com/TEAMMATES/teammates/blob/d521d365adbc619d02d7b32b631e1b17153ba353/src/test/resources/data/InstructorFeedbackResultsPageUiTest.json#L1683-L1694

Reply:
You are right. Seems legacy data for my example.